tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -383,73 +383,77 @@ if (/\/2\.9/.test(Cypress.env('rancher_version'))) {
-              cy.get('div.code-mirror.as-text-area').eq(1).click().type("test_data_value");
-              cy.clickButton('Add');
-            }
-            else if (resourceType === 'Services') {
-              cy.get("input[type=number]").clear().type("6341");
-            }
-
-            else  {
-              throw new Error(`Resource "${resourceType}" is invalid  / not implemented yet`);
-            }
-
-            cy.wait(500);
-            cy.clickButton('Save');
-            cy.filterInSearchBox(resourceName);
-            cy.verifyTableRow(0, resourceName, dataToAssert);
-            // Any mutable resource will reconcile to it's original state immediately
-            // But with ConfigMaps and Services it is not because they are immutable i.e.
-            // They didn't reconciled when `correctDrift` is used.
-            cy.deleteAllFleetRepos();
+          // Add GitRepo by enabling 'correctDrift'
+          cy.fleetNamespaceToggle('fleet-default')
+          cy.addFleetGitRepo({ repoName, repoUrl, branch, path, correctDrift: 'yes' });
+          cy.clickButton('Create');
+          cy.checkGitRepoStatus(repoName, '2 / 2');
+          cy.accesMenuSelection(dsClusterName, resourceLocation, resourceType);
+          cy.nameSpaceMenuToggle(resourceNamespace);
+          cy.filterInSearchBox(resourceName);
+          cy.get('.col-link-detail').contains(resourceName).should('be.visible');
+          cy.open3dotsMenu(resourceName, 'Edit Config');
+
+          if (resourceType === 'ConfigMaps') {
+            cy.clickButton('Add');
+            cy.get('[data-testid="input-kv-item-key-1"]').eq(0).focus().type('test_key');
+            cy.get('div.code-mirror.as-text-area').eq(1).click().type("test_data_value");
+            cy.clickButton('Add');
+          }
+          else if (resourceType === 'Services') {
+            cy.get("input[type=number]").clear().type("6341");
+          }
+
+          else  {
+            throw new Error(`Resource "${resourceType}" is invalid  / not implemented yet`);
+          }
 
-            // Delete leftover resources if there are any.
-            cy.accesMenuSelection(dsClusterName, resourceLocation, resourceType);
+          cy.wait(500);
+          cy.clickButton('Save');
+          cy.filterInSearchBox(resourceName);
+          cy.verifyTableRow(0, resourceName, dataToAssert);
+          // Any mutable resource will reconcile to it's original state immediately
+          // But with ConfigMaps and Services it is not because they are immutable i.e.
+          // They didn't reconciled when `correctDrift` is used.
+          cy.deleteAllFleetRepos();
+
+          // Delete leftover resources if there are any on each downstream cluster.
+          // Currently, service is getting deleted from cluster, hence adding check for it.
+          dsClusterList.forEach((dsClusterName) => {
+            cy.accesMenuSelection(dsClusterName, "Service Discovery", "Services");
```

Comment:
Suggestion: add small wait or any other page loaded check element that avoids interference with burgermenu
```
cy.wait(500);
```

Reply:
Added.